onnxruntime/core/graph/graph.cc

```diff
@@ -898,7 +871,12 @@ void Node::Init(std::string_view name,
   if (attributes) {
     attributes_ = *attributes;
 
+    isForwardNode_ = true;
```

Comment:
I am not particular happy tracking the kBackwardAttributeName at multiple places. Is there a chance to have a common pair of functions to add/remove new nodes to minimize breaking changes in the future?